cs-commons/src/main/java/io/cloudslang/content/utils/OtherUtilities.java

```diff
@@ -0,0 +1,61 @@
+    private static Pattern ipPattern = Pattern.compile("((^((([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5]))$)|(^((([0-9A-Fa-f]{1,4}:){7}([0-9A-Fa-f]{1,4}|:))|(([0-9A-Fa-f]{1,4}:){6}(:[0-9A-Fa-f]{1,4}|((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3})|:))|(([0-9A-Fa-f]{1,4}:){5}(((:[0-9A-Fa-f]{1,4}){1,2})|:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3})|:))|(([0-9A-Fa-f]{1,4}:){4}(((:[0-9A-Fa-f]{1,4}){1,3})|((:[0-9A-Fa-f]{1,4})?:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3}))|:))|(([0-9A-Fa-f]{1,4}:){3}(((:[0-9A-Fa-f]{1,4}){1,4})|((:[0-9A-Fa-f]{1,4}){0,2}:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3}))|:))|(([0-9A-Fa-f]{1,4}:){2}(((:[0-9A-Fa-f]{1,4}){1,5})|((:[0-9A-Fa-f]{1,4}){0,3}:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3}))|:))|(([0-9A-Fa-f]{1,4}:){1}(((:[0-9A-Fa-f]{1,4}){1,6})|((:[0-9A-Fa-f]{1,4}){0,4}:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3}))|:))|(:(((:[0-9A-Fa-f]{1,4}){1,7})|((:[0-9A-Fa-f]{1,4}){0,5}:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3}))|:)))(%.+)?$))", Pattern.CASE_INSENSITIVE);
+
+    private OtherUtilities() {}
+
+    /** Given an email string, it checks if it's a valid email with a regex (see emailPattern attribute)
+     * @param emailStr the string to check if it's a valid email
+     * @return true if it's a valid email, false otherwise
+     */
+    public static boolean isValidEmail(@NotNull final String emailStr) {
+        return emailPattern.matcher(emailStr).matches();
+    }
+
+    /** Given an ip port string, it checks if it's a valid ip port with a regex (see ipPortPattern attribute)
+     * @param portStr the string to check if it's a valid ip port
+     * @return true if it's a valid ip port, false otherwise
+     */
+    public static boolean isValidIpPort(@NotNull final String portStr) {
+        return ipPortPattern.matcher(StringUtils.strip(portStr)).matches();
+    }
+
+    /** Given an ip string, it checks if it's a valid ip with a regex (see ipPattern attribute)
+     * @param ipStr the string to check if it's a valid ip
+     * @return true if it's a valid ip, false otherwise
+     */
+    public static boolean isValidIp(@NotNull final String ipStr) {
+        return ipPattern.matcher(StringUtils.strip(ipStr)).matches();
+    }
+
+    /** Change the from windows line ending in unix line ending
+     * @param string the string in which to change the line ending
+     * @return a new string with the line ending changed from windows to unix
+     */
+    @NotNull
+    public static String changeNewLineFromWindowsToUnix(@NotNull final String string) {
+        return string.replaceAll("\r\n", "\n");
```

Comment:
most likely does not work, this expects a regex. You can use string.replace with `CharSequence`.